ssmconfig.go

```diff
@@ -6,14 +6,25 @@ import (
 	"path"
 	"reflect"
 	"strconv"
-
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ssm"
 	"github.com/aws/aws-sdk-go/service/ssm/ssmiface"
 	"github.com/pkg/errors"
 )
 
+type (
```

Comment:
Please move these type declarations back to their original places in the file. Making stylistic changes pollutes the diff.